lib/dalli/server.rb

```diff
@@ -26,12 +26,12 @@ class Server
       :value_max_bytes => 1024 * 1024,
       # surpassing value_max_bytes either warns (false) or throws (true)
       :error_when_over_max_size => false,
-      :compressor => Compressor,
+      :compressor => (defined?($TESTING) && $TESTING) ? Compressor : Dalli::Identity,
```

Comment:
Dalli's pattern seems to be omitting the shared module prefix.

```suggestion
      :compressor => (defined?($TESTING) && $TESTING) ? Compressor : Identity,
```

Reply:
We should arguably update the tests to pass `Compressor` instead of use the default, if they fail otherwise. I don't feel particularly strongly about that though.

Reply:
I tried that initially but it required a good amount of changes. I think we should probably keep the number of changes to a minimum to avoid diverging from upstream too much. Also I think doing it this way will allow us to bring in changes from upstream more easily in the future.